CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 - [IMPROVED] Updated `Getting started` section with a `get_query_result` example. 
 - [FIXED] Fixed parameter type of `selector` in docstring.
+- [IMPROVED] Removed internal `Document._document_id`property so dict's
```

Comment:
add a space: `Document._document_id`property ->  `Document._document_id` property

Reply:
I'm wondering if we should write something like:
```
[FIXED] Removed internal `Document._document_id` logic to keep dict' s`_id` field in sync with `_document_id` class parameter.
```
@smithsz wdyt?

Reply:
yeah, mark it as a `[FIXED]`.

Reply:
done, also rephrased the changelog entry so it's more clear